<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>786</width>
    <height>705</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="volume_label">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>190</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Volume</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="deviation">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>210</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="symbol_label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>190</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Symbol</string>
   </property>
  </widget>
  <widget class="QLabel" name="deviation_label">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>190</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Deviation</string>
   </property>
  </widget>
  <widget class="QPushButton" name="start_button">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>600</y>
     <width>351</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Start Bot Session</string>
   </property>
  </widget>
  <widget class="QLabel" name="sl_trail_label">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>260</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>SL trail amount</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="unit_pip">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>210</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="unit_pip_label">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>190</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Unit pip</string>
   </property>
  </widget>
  <widget class="QLabel" name="default_sl_label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>260</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Default SL</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="sl_trail">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>280</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="volume">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>210</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="default_tp">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>280</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="max_sl_dist">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>280</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="default_tp_label">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>260</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Default TP</string>
   </property>
  </widget>
  <widget class="QLabel" name="max_sl_dist_label">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>260</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Max SL distance</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="symbol">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>210</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="default_sl">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>280</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="login">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>110</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="password">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>110</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="server">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>110</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="login_label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>90</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Login</string>
   </property>
  </widget>
  <widget class="QLabel" name="password_label">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>90</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Password</string>
   </property>
  </widget>
  <widget class="QLabel" name="server_label">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>90</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Server</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="use_atr">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>350</y>
     <width>76</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Use ATR</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="save_config">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>660</y>
     <width>151</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Save configuration</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="sr_likelihood">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>510</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="sr_threshold">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>510</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="timeframe">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>210</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="strategy">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>510</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="strategy_label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>490</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Select strategy</string>
   </property>
  </widget>
  <widget class="QLabel" name="timeframe_label">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>190</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Seleect timeframe</string>
   </property>
  </widget>
  <widget class="QLabel" name="sr_likelihood_label">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>490</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>SR likelihood</string>
   </property>
  </widget>
  <widget class="QLabel" name="sr_thresold_label">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>490</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>SR threshold</string>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>7</x>
     <y>160</y>
     <width>771</width>
     <height>20</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="line_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>390</y>
     <width>771</width>
     <height>20</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLineEdit" name="atr_period">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>280</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="atr_period_label">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>260</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>ATR Period</string>
   </property>
  </widget>
  <widget class="QLabel" name="strategy_title">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>420</y>
     <width>361</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Strategy Cofiguration</string>
   </property>
  </widget>
  <widget class="QLabel" name="auth_title">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>30</y>
     <width>321</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Authentication Details</string>
   </property>
  </widget>
  <widget class="Line" name="line_3">
   <property name="geometry">
    <rect>
     <x>7</x>
     <y>560</y>
     <width>771</width>
     <height>20</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="help_button">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>660</y>
     <width>91</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Help</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="sr_period">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>510</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="sr_period_label">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>490</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>SR period</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="timezone_diff">
   <property name="geometry">
    <rect>
     <x>182</x>
     <y>350</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="timezone_diff_label">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>330</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Timezone difference</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="target_profit">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>349</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="max_loss">
   <property name="geometry">
    <rect>
     <x>502</x>
     <y>350</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="target_profit_label">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>330</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Target Profit (%)</string>
   </property>
  </widget>
  <widget class="QLabel" name="max_loss_label">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>330</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Maximum Loss (%)</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
